plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/related-by-control.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	CheckboxControl,
+	PanelBody,
+	// @ts-expect-error missing types.
+	__experimentalVStack as VStack,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { QueryControlProps, RelatedBy, CoreFilterNames } from '../../types';
+
+const RelatedByControl = ( {
+	query,
+	setQueryAttribute,
+	trackInteraction,
```

Comment:
Currently, we are only recording the props `collection`, `filter` and `location` under the event `blocks_product_collection_inspector_control_clicked`. However, I believe it would be beneficial to also record whether `Categories` or `Tags` are selected. Perhaps we can track this under a separate event, such as `blocks_product_collection_inspector_control_related_selected` or something similar.